suite-native/module-add-accounts/src/screens/AddCoinDiscoveryRunningScreen.tsx

```diff
@@ -0,0 +1,104 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { networks, NetworkSymbol } from '@suite-common/wallet-config';
+import { VStack, Text, Spinner, SpinnerLoadingState } from '@suite-native/atoms';
+import { Screen } from '@suite-native/navigation';
+import { Translation } from '@suite-native/intl';
+import { Account } from '@suite-common/wallet-types';
+import {
+    AccountsRootState,
+    DeviceRootState,
+    selectDeviceAccountsByNetworkSymbol,
+    selectDeviceDiscovery,
+} from '@suite-common/wallet-core';
+import { toggleEnabledDiscoveryNetworkSymbol } from '@suite-native/discovery';
+import { useCoinEnabling } from '@suite-native/coin-enabling';
+
+import { useAddCoinAccount } from '../hooks/useAddCoinAccount';
+
+export type AddCoinDiscoveryRunningScreenProps = {
+    networkSymbol: NetworkSymbol;
+};
+
+export const AddCoinDiscoveryRunningScreen = ({ route }) => {
+    const { networkSymbol, flowType } = route.params;
+    const dispatch = useDispatch();
+    const accounts = useSelector((state: AccountsRootState & DeviceRootState) =>
+        selectDeviceAccountsByNetworkSymbol(state, networkSymbol),
+    );
+    const normalAccounts = accounts.filter(a => a.accountType === 'normal');
+    const nonEmptyAccounts = accounts.filter(a => !a.empty);
+
+    const discovery = useSelector(selectDeviceDiscovery);
+    const { navigateToAddCoinDiscoveryFinishedScreen, navigateToSuccessorScreen } =
+        useAddCoinAccount();
+    const { enabledNetworkSymbols, applyDiscoveryChanges } = useCoinEnabling();
```

Comment:
```suggestion
    const { applyDiscoveryChanges } = useCoinEnabling();
    const enabledNetworkSymbols = useSelector(selectEnabledDiscoveryNetworkSymbols);

```
WHy is it needed to access the symbols from a custom hook if it is possible to access it directly from the state? I think this adds an extra complexity again.

Reply:
For me it actually helps with readability and ensures that correct selectors are used everytime we want to work with coin enabling. 

Lets discuss + and -  of this on Monday 🙏

Reply:
IMO the selecetor should be named in the way you always know which one you want to use.  Then this is just extra abstraction. But I'm not gonna push it. Up to you.

Reply:
I'll refactor this as we spoke

Reply:
6467982f20